recipes/amazon-braket-default-simulator/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "amazon-braket-default-simulator" %}
+{% set version = "1.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/amazon-braket-default-simulator-{{ version }}.tar.gz
+  sha256: 9880d8044e49415a6e4c862314a23277915b7de1c6a236992df255f31c2f550f
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
Not `noarch` due to https://github.com/aws/amazon-braket-default-simulator-python/blob/8b3e7478abca239eb6fd68d4ce4c41cee6bfd49d/setup.py#L30-L36

Reply:
I am unsure that this is a problem... let me check.

Reply:
The conda-forge knowledge base says:

> Note Only console_scripts entry points have to be listed in meta.yaml. Other entry points do not conflict with noarch and therefore do not require extra treatment.

Reply:
Oh, yeah you're right, I completely misremembered that. Thanks!